src/createServer.ts

```
@@ -13,6 +13,8 @@ import {
 } from "./envConfigSchema";
 import plugins from "./plugins/index";
 import routes from "./routes/index";
+import { IncomingHttpHeaders } from "http";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)